instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/ConfigPropertiesUtil.java

```diff
@@ -17,11 +17,25 @@
  */
 public final class ConfigPropertiesUtil {
```

Comment:
I think you had renamed this to Legacy something? The reason I’m interested is because worried people will propagate the pattern of using it to new places.

Reply:
no, it was a different class that I've removed now.

`ConfigPropertiesUtil` is used both for this legacy use case and for early init properties. I've opted to add javadoc comments to discourage use of this class.